rust/stracciatella_c_api/cbindgen.toml

```diff
@@ -1,12 +1,39 @@
 autogen_warning = "/* DO NOT MODIFY MANUALLY! Changes will be discarded during build. */"
-include_guard = "stracciatella_h"
 include_version = true
 language = "C++"
+sys_includes = [
+    # std::unique_ptr
+    "memory",
+]
+
+# Top of stracciatella.h
+header = '''
+#pragma once'''
```

Comment:
Currently `cbindgen` has no way to place extra code inside the guard.

I didn't opt for an extra guard wraping the new code (one of the alternatives) because we already rely on `#pragma once` in the C++ code.

If we want to avoid it, we should also remove it from the C++ code.
We target C++11 and always use it in the same filesystem so I think it's fine to rely on it.